Makefile

```diff
@@ -9,6 +9,7 @@ GO_FILES = $(shell find . -type f -name '*.go')
 .PHONY: all
 all: clean exo
 
+.PHONY: exo
```

Comment:
It shouldn't be phony because it creates an executable called `exo`, should it?

Reply:
Good point. Fixed.